runtime/queries/sql/highlights.scm

```diff
@@ -34,10 +34,10 @@
 (marginalia) @comment.block
 
 ((literal) @constant.numeric.integer
-   (#match? @constant.numeric.integer "^[-+]?\\d+$"))
+   (#match? @constant.numeric.integer "^[-+]?%d+$"))
 
 ((literal) @constant.numeric.float
-  (#match? @constant.numeric.float "^[-+]?\\d*\\.\\d*$"))
+  (#match? @constant.numeric.float "^[-+]?%d*\.%d*$"))
```

Comment:
I believe the percents here are from `#lua-match?` in nvim-treesitter. The lua regex engine I believe replaces some escapes like `\d` with `%d`. If this is adapted from upstream we could suggest replacing with `[0-9]` there for portability. I think for now it should be enough to reset these blocks